app-cryptography/signing-party/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=signing-party
+PKGSEC=misc
+PKGDES="Various OpenPGP related tools made for key signing party"
+PKGDEP="gnupg perl-class-methodmaker perl-mailtools perl-mime-tools perl-net-idn-encode perl-term-readkey perl-text-template \
+	perl-gnupg-interface imagemagick python-3 perl-gd qrencode qprint"
+BUILDDEP="perl gnupg"
+# TeXLive: for LaTeX support in gpg-key2latex.
+# Postfix: for mailing the encrypted messages containing signatures
+PKGSUG="texlive postfix"
+
+ABSPLITDBG=0
```

Comment:
`noarch` or no-split?

Reply:
It contains some C code, but noting got stripped out.

Reply:
> It contains some C code, but noting got stripped out.

That means this is a problem with passing in our Autobuild3 flags - this needs to be addressed.